utils/formatDate.ts

```diff
@@ -16,12 +16,15 @@ import { isValid } from 'date-fns'
 import { format, toDate, utcToZonedTime } from 'date-fns-tz'
 import enUS from 'date-fns/locale/en-US'
 import es from 'date-fns/locale/es'
+import { DateTime, Settings } from 'luxon'
 
 import { ApiGatewayDateString } from '../types/common'
 
 const pacificTimeZone = 'America/Los_Angeles'
 const apiGatewayFormat = "yyyy-MM-dd'T'HH:mm:ss"
 
+Settings.defaultZone = 'America/Los_Angeles'
```

Comment:
Nit:

```suggestion
Settings.defaultZone = pacificTimeZone
```

Reply:
good nit - but we end up not using the `pacificTimeZone` anywhere else so I just delete it in the follow up!